docs/examples/2048/2048_full.yaml

```diff
@@ -48,8 +48,8 @@ metadata:
   annotations:
     alb.ingress.kubernetes.io/scheme: internet-facing
     alb.ingress.kubernetes.io/target-type: ip
+    kubernetes.io/ingress.class: alb
 spec:
-  ingressClassName: alb
```

Comment:
This change was intentional

Reply:
@kishorj Thanks for the review. I understand it may be intentional, however when running that manifest as is, the error I posted originally is what is produced. I am running latest EKS cluster version and ALB controller version.

Reply:
Adding additional comment: I recreated my eks cluster and load balancer controller using the additional helm switch you recommended. I still receive this error:
Error from server (InternalError): error when creating "2048_full.yaml": Internal error occurred: failed calling webhook "vingress.elbv2.k8s.aws": Post "https://aws-load-balancer-webhook-service.kube-system.svc:443/validate-networking-v1beta1-ingress?timeout=10s": no endpoints available for service "aws-load-balancer-webhook-service"

Reply:
Could you check if the aws-load-balancer-controller pods are up and running? If you still have the problem, you can open a new issue. I will close this PR.